### DISCUSSION TITLE
How to sign the data use agreement for the project?

### DISCUSSION THREAD

**Discussion by M7m0d** [MIT-LCP/mimic-code#1640] [MIMIC-General] [answered]
I already completed the required training, and the status of my training is "Active".

Now, I need to sign the data use agreement for the project, but when I click on the link of "sign the data use agreement" I get this message: "In order to access a restricted-access resource, you must first be a [credentialed user](https://physionet.org/settings/credentialing/)."

My questions are:
1. Does that mean I should apply for the credential before I sign the data use agreement for the project?
2. What is the next step after completing the required training to access the dataset (known that the training is "Active")?

I hope my questions are clear. Thank you in advance.

![image](https://github.com/MIT-LCP/mimic-code/assets/37560169/39c44f92-596d-45ba-a93a-a274e597375f)

**Comment by tompollard** [comment, marked as answer]
> Does that mean I should apply for the credential before I sign the data use agreement for the project?

Yes, you will need to complete the credentialing process in order to gain access to the data. You can do this here: https://physionet.org/settings/credentialing/. 

> What is the next step after completing the required training to access the dataset 

Once your identity is confirmed (i.e. you are "credentialed") and any training requirements are met, signing the Data Use Agreement is just a quick click of a button.

**Reply by M7m0d** [reply]
Thank you very much for your help. I really appreciate it. :)
